### DISCUSSION TITLE
browser.keyboard.down & browser.keyboar.up raises an error

### DISCUSSION THREAD

**Discussion by KutsOleksii** [rubycdp/ferrum#274] [Q&A]
/usr/local/bundle/gems/ferrum-0.11/lib/ferrum/keyboard.rb:35:in `down': no implicit conversion of Symbol into Integer (TypeError)
It rises in this code:

    browser.keyboard.down(:pagedown)

any other key including "a" raises the same error
this is line 35 in keyboard.rb

      type = key[:text] ? "keyDown" : "rawKeyDown"
========================================
/usr/local/bundle/gems/ferrum-0.11/lib/ferrum/keyboard.rb:42:in `up': no implicit conversion of Array into Hash (TypeError)
It rises in this code:
    browser.keyboard.up(:pagedown)
any other key including "a" raises the same error
this is line 42 in keyboard.rb

      @page.command("Input.dispatchKeyEvent", slowmoable: true, type: "keyUp", **key)

works only 'type' method. 'up' and 'down' raises  an error.

Did i miss something?

**Comment by route** [comment]
Yea looks broken, I created a PR https://github.com/rubycdp/ferrum/pull/288
